.github/workflows/push.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         uses: github/codeql-action/analyze@v2
 
       - name: run fastfetch
-        run: ./fastfetch --recache --disable-linewrap false --hide-cursor false --show-errors true
+        run: ./fastfetch --recache --disable-linewrap false --hide-cursor false --show-errors true --load-config ../presets/all
```

Comment:
GH actions builds in the source code directory, so the path is just `presets/all`

Reply:
fixed